src/Servers/Kestrel/Core/src/KestrelServerOptions.cs

```diff
@@ -72,6 +72,11 @@ public class KestrelServerOptions
         /// </summary>
         public KestrelConfigurationLoader ConfigurationLoader { get; set; }
 
+        /// <summary>
+        /// Controls whether to return the AltSvcHeader from on an HTTP/2 or lower response for HTTP/3
+        /// </summary>
+        public bool DisableAltSvc { get; set; } = true;
```

Comment:
EnableAltSvc (defaulting to false)

Reply:
We'll need to figure out a way to automatically configure this to sending altsvc if HTTP/2 and HTTP/3 are enabled at the same time.

Reply:
But in general, boolean properties should be named in the positive (`Enable` not `Disable`, etc.). So `EnableAltSvc` with a default value of `false` (to be disabled by default). We can figure out the automatic configuration later.